### PR TITLE
feat(website): added maci wrapper quickstarter in the docs

### PR DESCRIPTION
**Pull request by yashgo0018**
# Description

Added MACI Wrapper Quickstarter guide in the docs.

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/versioned_docs/version-v2.0_alpha/developers-references/maci-wrapper/QuickStartGuide.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
+---
+title: MACI Wrapper Guide Scaffold ETH 2 + MACI Voting Template
+description: Introduction to the MACI Wrapper and Setup
+sidebar_label: Quickstart Guide
+sidebar_position: 1
+---
+
+# MACI Wrapper Guide: Scaffold ETH 2 + MACI Voting Template
+
+Welcome to the hacker guide for the Scaffold ETH 2 + MACI Voting Template! This guide will help you get started with the template, providing step-by-step instructions to set up and customize your decentralized voting application, understand MACI, and explore project ideas.
```
Comment (ctrlc03) [inline website/versioned_docs/version-v2.0_alpha/developers-references/maci-wrapper/QuickStartGuide.md:10]: let's add here a link to the repo and add your name + link as author (if you wish to) 
something like
[Scaffold ETH 2 + MACI Voting Template](https://github.com/yashgo0018/maci-wrapper) by [Yash](https://github.com/yashgo0018)  ?

**Comment by yashgo0018** [comment]
Added the link and author info

File: website/versioned_docs/version-v2.0_alpha/developers-references/maci-wrapper/QuickStartGuide.md (hunk 1 above)
Comment (yashgo0018) [reply website/versioned_docs/version-v2.0_alpha/developers-references/maci-wrapper/QuickStartGuide.md:10]: Added the link and the author info

**Review by ctrlc03 — APPROVED** [review]
Thank you

**Pull request merged**